nekoyume/Assets/_Scripts/UI/Widget/QuestPreparation.cs

```diff
@@ -501,7 +501,7 @@ private void ReadyToQuest(bool ready)
                 }
             }
 
-            questButton.SetConditionalState(ready);
+            questButton.UpdateObjects();
```

Comment:
I think this line should be in front of line 494. `IsSubmittable` is checked before checking condition.